src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![deny(warnings)]
-
+#![allow(unused)]
```

Comment:
This was removed because it's **really** dangerous, it allows unused results for example.
Please don't make me do another PR to fix everything two or three months from now ;)

Reply:
;;)